library/Vanilla/Formatting/Formats/MarkdownFormat.php

```diff
@@ -47,10 +47,30 @@ public function __construct(
         }
     }
 
+    /**
+     * This copy of legacy spoilers processes first.
+     *
+     * @param string $html The HTML to process.
+     * @return string
+     */
+    private function parentLegacySpoilers(string $html): string {
+        return parent::legacySpoilers($html); // TODO: Change the autogenerated stub
+    }
+
+    /**
+     * This override does not format spoiler so that it can be done early.
+     *
+     * @inheritdoc
+     */
+    protected function legacySpoilers(string $html): string {
+        return $html;
+    }
+
     /**
      * @inheritdoc
      */
     public function renderHtml(string $value, bool $enhance = true): string {
+        $value = $this->parentLegacySpoilers($value);
```

Comment:
Since `parentLegacySpoilers` is private, and we're currently only using it in this one place, what's the primary benefit of the separate method versus just making the call to `parent::legacySpoilers` here?

Reply:
That is a brain fart. I'll change.